scaleway/resource_container_namespace.go

```diff
@@ -145,8 +145,7 @@ func resourceScalewayContainerNamespaceUpdate(ctx context.Context, d *schema.Res
 	}
 
 	if d.HasChange("description") {
-		description := d.Get("description").(string)
-		req.Description = &description
+		req.Description = expandUpdatedStringPtr(d.Get("description"))
```

Comment:
Should we add an `hasChanged = true` here?

Reply:
Only one update request can be made in this function, if we entered it, it is because at least of field has changed anyway